vars/withInfraPipeline.groovy

```diff
@@ -30,6 +30,7 @@ def call(String product, String component = null, Closure body) {
 
   node(agentType) {
     def slackChannel = env.BUILD_NOTICES_SLACK_CHANNEL
+    def expiresAfter = env.EXPIRES_AFTER
```

Comment:
this won't necessarily be set, you should probably get it from the `pipelineConfig` and it should be initialised to a value there as well

probably something like:
```suggestion
    def expiresAfter = pipelineConfig.infraExpiresAfterTag
```